### PR TITLE
 Added more filters

### LINKED ISSUE #234: Add more filters in filter images screen
This is a list of filters that can be applied using PhotoEditor library we are using:
https://github.com/burhanrashid52/PhotoEditor/blob/b32e40c091f336d4123da45da901feacb66972fd/photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoFilter.java

We currently have only 2 filters : None, black & white, sepia
screen shot 2018-07-16 at 7 40 09 pm

Add the filters listed in above link also to the application

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #234

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ x] `./gradlew assembleDebug assembleRelease`
- [x ] `./gradlew checkstyle`

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java [hunk 1]
```diff
@@ -260,8 +260,27 @@ private void previous() {
      */
     private void showFilters() {
         mFilterItems.add(new FilterItem(R.drawable.none, "None"));
+        mFilterItems.add(new FilterItem(R.drawable.auto_fix, "Auto Fix"));
         mFilterItems.add(new FilterItem(R.drawable.black, "GrayScale"));
+        mFilterItems.add(new FilterItem(R.drawable.brightness, "Brightness"));
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: Add the constant strings in here to strings.xml file & use those here.

**Comment by shiv07tiwari** [comment]
screenshot_20180721_094930
@Swati4star

**Comment by Swati4star** [comment]
The filter images were circular before. You have made them square.

**Comment by shiv07tiwari** [comment]
I just changed the background color because for every effect, the background was balck(in newly added ones)

**Comment by shiv07tiwari** [comment]
screenshot_20180721_100615
New UI @Swati4star

**Pull request merged**